test/parallel/test-regress-GH-12593.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
Could you add this to an appropriate existing test or give the file a better name. `regress-GH-*.js` names aren't really helpful for people working on the tests in the future.